src/Event/Event.php

```diff
@@ -89,6 +94,29 @@ public function __get($attribute)
         if ($attribute === 'name' || $attribute === 'subject') {
             return $this->{$attribute}();
         }
+        if ($attribute === 'data') {
+            return $this->_data;
+        }
+        if ($attribute === 'result') {
+            return $this->_result;
+        }
+    }
+
+    /**
+     * Provides backward compatibility for write access to data and result properties.
+     *
+     * @param string $attribute Attribute name.
+     * @param mixed $value The value to set.
+     * @return void
```

Comment:
Should these methods be marked as 'deprecated'. With all the new reader methods, I don't think we need to have setters/magic get.